### ISSUE TITLE
File system initialization step 3 failure (Windows 8 64 bit, Chrome 64 bit)

### ISSUE THREAD

**Issue by Corecii** [vladikoff/chromeos-apk#48]
**TL;DR:** ARChon cannot finish initializing the file system. It crashes on step 3. All information is below in order to confirm that I didn't mess it up in the process, as well as important logs. I'll give any more information needed to help determine the source of this error.

Canary version: 39.0.2167.0 (Official Build) canary

I attempted to load the ARChon runtime and the 2048 app on Chrome stable, Chrome beta, and Chrome Canary. My results have lead me only to failure. I have also attempted the same a few days ago with version 1.0 and got the same results.
Before continuing:
I have followed all instructions. ARChon unzipped, installed unpacked, loaded first. App installed the same way afterwards. 'Launch' button clicked. I've also done some fiddling and retrying to make sure it wasn't an error of my own.
All tests were done on x64 chrome on an x64 computer with x64 Windows 8 (not 8.1) with the x64 version of ARChon and the provided 2048 app.
Chrome://version was checked before each test to make sure the version was as it was expected. Full exit out of Chrome to switch from Beta to Stable.
ARChon version 1.1 was used for these tests. ARChon version 1.0 was tested previously a few days ago with the same file system problem.

Chrome Canary x64: I was unable to find this in the log for the page on inspecting app_main.html. I did, on the other hand, find it on the extensions page's log. I'm not sure why I was unable to find it in the log for app_main.html but the error is still there and is still relevant. I'd previously found it in the app_main.html log for ARChon 1.0 a few days back as well.

![](http://i.imgur.com/vXuMaN4.png)

Both Chrome Beta x64 and Chrome Stable x64 appear to not be 'new enough' for ARChon. The information is included in case it may be important.

Chrome Beta x64: chrome.appViewInternal is not available: 'appViewInternal' requires Google Chrome dev channel or newer, but this is the beta channel. app_main.html:1
Full log: http://i.imgur.com/StZmCf4.png

Chrome Stable x64: Re-installed to stable, same error as beta, but 'beta channel' = 'stable channel'
Full log: http://i.imgur.com/kMU25Ia.png

If there's any more information I should give, or some log file (rather than the console) I should be getting information from, I'd be happy to give it. This appears to be the same issue that first appeared in https://github.com/vladikoff/chromeos-apk/issues/10 but it was never addressed in that issue and discussion moved on to other matters.

A little bit more information:
While it's obvious the file system initialization is causing the crash, what's not obvious is that it may have simply not printed the 3rd part and crashed sometime later. I have fiddled with the code and it definitely crashes on step 3. All tests above were done with unmodified code.
